src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/_params.py

```diff
@@ -0,0 +1,170 @@
+        c.argument('requires_session', options_list=['--requires-session'], action='store_true', help='A boolean value that indicates whether the queue supports the concept of sessions.')
+        c.argument('default_message_time_to_live', options_list=['--default-message-time-to-live'], validator=_validate_default_message_time_to_live, help='ISO 8601 default message timespan to live value. This is the duration after which the message expires, starting from when the message is sent to Service Bus. This is the default value used when TimeToLive is not set on a message itself.')
+        c.argument('dead_lettering_on_message_expiration', options_list=['--dead-lettering-on-message-expiration'], action='store_true', help='A boolean value that indicates whether this queue has dead letter support when a message expires.')
+        c.argument('duplicate_detection_history_time_window', options_list=['--duplicate-detection-history-time-window'], validator=_validate_duplicate_detection_history_time_window, help='ISO 8601 timeSpan structure that defines the duration of the duplicate detection history. The default value is 10 minutes.')
+        c.argument('max_delivery_count', options_list=['--max-delivery-count'], type=int, help='The maximum delivery count. A message is automatically deadlettered after this number of deliveries. default value is 10.')
+        c.argument('status', options_list=['--status'], arg_type=get_enum_type(['Active', 'Disabled', 'Restoring', 'SendDisabled', 'ReceiveDisabled', 'Creating', 'Deleting', 'Renaming', 'Unknown']), help='Enumerates the possible values for the status of a messaging entity.')
+        c.argument('auto_delete_on_idle', options_list=['--auto-delete-on-idle'], validator=_validate_auto_delete_on_idle, help='ISO 8601 timeSpan idle interval after which the queue is automatically deleted. The minimum duration is 5 minutes.')
+        c.argument('enable_partitioning', options_list=['--enable-partitioning'], action='store_true', help='A boolean value that indicates whether the queue is to be partitioned across multiple message brokers.')
+        c.argument('enable_express', options_list=['--enable-express'], action='store_true', help='A boolean value that indicates whether Express Entities are enabled. An express queue holds a message in memory temporarily before writing it to persistent storage.')
+        c.argument('forward_to', options_list=['--forward-to'], arg_type=name_type, help='Queue/Topic name to forward the messages')
+        c.argument('forward_dead_lettered_messages_to', arg_type=name_type, options_list=['--forward-dead-lettered-messages-to'], help='Queue/Topic name to forward the Dead Letter message')
+
+    # region Queue Authorizationrule
+    with self.argument_context('servicebus queue authorizationrule') as c:
```

Comment:
`authorizationrule` should be hyphenated: `authorization-rule`.

Reply:
implemented